train-simulation/simulator/config/node-red/entrypoint.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
 
+# Commit of node-red.trainsim repository
+TRAINSIM_COMMIT="b4a68f144e33fba3b053ff92503e352426f2fdc8"
```

Comment:
Can we inject the commit id via docker-compose.yaml? It more obvious then

Reply:
Good idea.